src/main.rs

```diff
@@ -176,12 +176,13 @@ fn run() -> Result<ExitCode> {
             workflow = workflow.filename().cyan()
         ));
         for (name, audit) in audit_registry.iter_workflow_audits() {
-            results.extend(audit.audit(workflow).with_context(|| {
+            let findings = audit.audit(workflow).with_context(|| {
                 format!(
                     "{name} failed on {workflow}",
                     workflow = workflow.filename()
                 )
-            })?);
+            })?;
+            results.extend(findings);
```

Comment:
Is this new `findings` variable needed?

Reply:
Definetely not needed! Just a leftover from the first take 🙈

Reply:
Solved [here](https://github.com/woodruffw/zizmor/pull/187/commits/0de3177ed5dd2f3e944b3e2edc527742e4531a5e)